src/response.rs

```diff
@@ -96,13 +95,13 @@ impl Response {
 
     /// This functions return a Result with a [ParsedGraphqlResponse]
     /// Look at the type documentation for more specifications
-    pub fn get_graphql_response<T>(&self) -> PrimaBridgeResult<ParsedGraphqlResponse<T>>
+    pub fn parse<T>(&self) -> PrimaBridgeResult<ParsedGraphqlResponse<T>>
```

Comment:
Maybe parse is too generic for this? Isn't it better to use something like `parse_grapqhl_response`?